client/client_test.go

```diff
@@ -629,12 +644,13 @@ func testHostnameSpecifying(t *testing.T, sb integration.Sandbox) {
 
 // moby/buildkit#614
 func testStdinClosed(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	c, err := New(sb.Context(), sb.Address())
 	require.NoError(t, err)
 	defer c.Close()
 
-	st := llb.Image("busybox:latest").Run(llb.Shlex("cat"))
+	imgName := integration.UnixOrWindows("busybox:latest", "nanoserver:latest")
```

Comment:
how about , `imgName := integration.UnixOrWindows("busybox", "nanoserver") + ":latest"`

Reply:
sure, as in #5574